XSUP-66843_add_azure_resourceGraph_gov_support (#43805)
demisto/content: Packs/AzureResourceGraph/ReleaseNotes/1_0_14.md

#### Integrations

##### Azure Resource Graph

> - Added support for **Use Gov Account Settings** parameter that select this checkbox if you are using a us government cloud account.

Reviewer: ```suggestion
Added the *Use Gov Account Settings* parameter to support US Government cloud accounts.
```
